holoviews/plotting/bokeh/plot.py

```diff
@@ -349,23 +350,50 @@ class CompositePlot(BokehPlot):
     to such a plot.
     """
 
-    fontsize = param.Parameter(default={'title': '16pt'}, allow_None=True,  doc="""
+    fontsize = param.Parameter(default={'title': '15pt'}, allow_None=True,  doc="""
        Specifies various fontsizes of the displayed text.
 
        Finer control is available by supplying a dictionary where any
        unmentioned keys reverts to the default sizes, e.g:
 
           {'title': '15pt'}""")
 
-    _title_template = "<span style='font-size: {fontsize}'><b>{title}</b></font>"
+    _title_template = (
+        '<span style='
+        '"color:{color};font-family:{font};'
+        'font-style:{fontstyle};font-weight:{fontstyle};'  # italic/bold
+        'font-size:{fontsize}">'
+        '{title}</span>'
+    )
 
     def _get_title(self, key):
         title_div = None
         title = self._format_title(key) if self.show_title else ''
         if title:
```

Comment:
To avoid the nesting I'd change this to:

```python
if not title:
    return title_div
```

and then un-indent the rest of the code in this section.